### PR TITLE
[bitnami/nginx] Use the new helper for HPA API version

### PR DESCRIPTION
**Pull request by miguelaeh**
### Applicable issues

  - fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by FraPazGal — APPROVED** [review]

File: bitnami/nginx/templates/hpa.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: {{ include "common.capabilities.hpa.apiVersion" . }}
```
Comment (kirkvogen) [inline bitnami/nginx/templates/hpa.yaml:2]: I think that we may have a conflict with this change and the changes on lines 26-27. I put a [comment on the commit](https://github.com/bitnami/charts/commit/624e1bffc0d29b45d5d09c56ef35b4ea1e485f62#r73791839) about it.

Comment (miguelaeh) [reply bitnami/nginx/templates/hpa.yaml:2]: It was fixed, thanks!

Comment (FraPazGal) [reply bitnami/nginx/templates/hpa.yaml:2]: Thanks for the heads up! We took notice of this issue and addressed it in a later PR, https://github.com/bitnami/charts/pull/10332

Comment (kirkvogen) [reply bitnami/nginx/templates/hpa.yaml:2]: I tried it out in chart version 11.1.5 and it works now with a server version less than 1.23.0. Thanks!

(The issue was originally seen in version 11.1.2)

**Pull request merged**